intro-OpenStack/ch_Shares.tex

```diff
@@ -0,0 +1,112 @@
+  ``public'', making it visible for any other project in the VSC cloud
+  as well.
+\item Click \textbf{Create} to complete this step.
+\end{enumerate}
+% https://docs.openstack.org/security-guide/shared-file-systems/share-acl.html
+
+At this point, the shared file system exists within OpenStack, but it cannot be used until we define access rules for it.
+
+\section*{Defining \gls{nfs} access rules}\label{sec:defin-nfs-access}
+You must define rules that define which machines on the network may
+obtain read or write access to your shared file system.  By default,
+in absence of any rules, a shared file system cannot be accessed by
+anyone.
+
+%TODO screenshots?
+\begin{enumerate}
+\item Open the drop-down menu in the \textbf{Actions} column for your share, and click \textbf{Manage Rules}.
+\item You can now see all Share Rules for this shared file system.
+  For a newly created file system, the list will be empty.  Click \textbf{Add rule}.
+\item Fill out the \textbf{Add Rule} dialog:
+  \begin{description}
+  \item[Access Type] Only ``ip'' is supported.
+  \item[Access Level] Choose if you want to give read and write
+    (``rw'') or read-only (``ro'') permission with this rule.
+  \item[Access To] Here, you can specify an ip address, or an address
+    range, to which the rule applies.  The addresses should be
+    specified according to the format expected by an NFS exports
+    configuration file.  The following table contains a few examples,
+    where it is assumed that the project's \_nfs network has the
+    address range 10.10.x.0/24:
+
+    \begin{tabular}{>{\bfseries}lp{0.7\textwidth}}
+      10.10.x.13 & Allow this single ip address.
+      \\ \hline
+      0.0.0.0/0 & Allow any ip address.
+      \\ \hline
+      10.10.x.0/24 & Allow any ip address from the project's \_nfs network.  For a non-public shared file system this has the same effect as the previous rule, because such a shared file system can only be accessed from within our project's \_nfs network anyway.
+      \\ \hline 
+      10.10.x.0/28 & Allow addresses 10.10.x.0 until 10.10.x.15.
+      \\
+    \end{tabular}
+
+  \end{description}
+  Click \textbf{Add} to add the rule.
+\end{enumerate}
+Your rule now appears in the list.  You can add as many rules as you
+wish, to set the access level for different addresses or address
+ranges.
+
+\section*{Accessing a shared file system}\label{sec:access-shar-file}
+When the proper access rules for the shared file system are in place,
+you can access it from an instance with a matching ip.  In order to be
+able to mount the shared file system, your instance needs
+\begin{itemize}
+\item a \gls{nfs} client, installed by default on images provided by
+  the VSC cloud, and
+\item access to the \_nfs network.  Because your instance likely has to
+  connect to the \_vm network as well, your VM should have two
+  \gls{nic}'s.  Again, this is taken care of in the default images.
+\end{itemize}
```

Comment:
there is a 3rd requirement, but @alvarosimon  should add  it to the default /standard images:
the idmap file need a VSC config.
can you add this an advanced requirement (or something like that, as it will only affect people who use their own images)

Reply:
@stdweird about the idmaping  it was disabled in our NFS server, so we do not need to include this info in our documentation I guess...

About our Images properties, if they want to upload their own images they should add the property `hw_vif_multiqueue_enabled=true`    to enable the Virtio network queues (not sure if this is too much info for a regular user). 
We can include the OpenStack link: https://docs.openstack.org/glance/latest/admin/useful-image-properties.html to provide more info regarding this.

This is set to `TRUE`  in all our public images at this moment.